src/components/Account.vue

```diff
@@ -5,28 +5,28 @@
       <h1>Your profile</h1>
     </div>
     <p v-if="profile.name">
-      <strong>Name:</strong> {{profile.title}} {{profile.name}}
+      <strong>Name:</strong> {{ profile.title }} {{ profile.name }}
     </p>
   </div>
 </template>
 
 <style lang="scss" scoped>
-  .headline{
-    margin-top: 1em;
-    display: flex;
-    img {
-      height: 80px;
-      width: 80px;
-      border-radius: 50%;
-    }
+.headline {
+  margin-top: 1em;
+  display: flex;
+  img {
+    height: 80px;
+    width: 80px;
+    border-radius: 50%;
   }
+}
 </style>
```

Comment:
There is an experiment here with the 'prettier' plugin. It automatically "fixes" code formatting. It will not typically be so noisy. This is its first run.